cmd/osv-scanner/scan/main.go

```diff
@@ -1,383 +1,29 @@
 package scan
 
 import (
-	"errors"
-	"fmt"
 	"io"
-	"net/http"
-	"os"
-	"os/exec"
-	"path/filepath"
-	"runtime"
-	"slices"
-	"strings"
-	"time"
 
-	"github.com/google/osv-scanner/internal/spdx"
-	"github.com/google/osv-scanner/pkg/models"
-	"github.com/google/osv-scanner/pkg/osvscanner"
+	"github.com/google/osv-scanner/cmd/osv-scanner/scan/docker"
+	"github.com/google/osv-scanner/cmd/osv-scanner/scan/project"
 	"github.com/google/osv-scanner/pkg/reporter"
-	"golang.org/x/term"
 
 	"github.com/urfave/cli/v2"
 )
 
-// flags that require network access and values to disable them.
-var offlineFlags = map[string]string{
-	"skip-git":                             "true",
-	"experimental-offline-vulnerabilities": "true",
-	"experimental-no-resolve":              "true",
-	"experimental-licenses-summary":        "false",
-	// "experimental-licenses": "", // StringSliceFlag has to be manually cleared.
-}
+const projectSubCommand = "project"
+
+const DefaultSubcommand = projectSubCommand
+
+var Subcommands = []string{projectSubCommand, "docker"}
```

Comment:
do future proof this, should instead make this "osv-scanner scan container" as opposed to docker?

Reply:
Let's change it to `osv-scanner scan image`